packages/bits/src/lib/tabgroup/tab-heading/tab-heading.component.ts

```diff
@@ -29,17 +34,29 @@ export class TabHeadingComponent {
         this.changeDetector.detectChanges();
     }
 
-    get active() {
-        return this._active;
-    }
-
     /** Tab id */
     @Input() tabId: string;
 
     /** Event is fired when tab became active, $event:Tab equals to selected instance of Tab component */
     @Output() selected: EventEmitter<TabHeadingComponent> = new EventEmitter();
 
-    protected _active: boolean;
+    @ViewChild("tab") tabEl: ElementRef<HTMLElement>;
+
+    @HostListener("keydown", ["$event"])
+    onKeyDown(event: KeyboardEvent): void {
+        const { key } = event;
```

Comment:
Try `code` instead of  `key`. Seems like for the "Space" key the `key` property returns an empty string. Currently, the "Space" navigation does not work

Reply:
thanks for correction, you are right.